src/core_atmosphere/physics/mpas_atmphys_camrad_init.F

```diff
@@ -822,12 +822,16 @@ subroutine oznini(mesh,atm_input)
  do j=1,latsiz ! latsiz=64
  do k=1,levsiz ! levsiz=59
  do i=1,lonsiz ! lonsiz=1
-    read(oz_unit,*) ozmixin(i,k,j,m)
+    read(read_unit,*) ozmixin(i,k,j,m)
  enddo
  enddo
  enddo
  enddo
- close(oz_unit)
+ close(read_unit)
+
+ if(dminfo % my_proc_id == IO_NODE) then
+    call mpas_release_unit(read_unit)
```

Comment:
If we let all MPI ranks read the file (which I would argue is not best practice, but it's what has apparently been working without issue so far), then I think `mpas_release_unit` should be called by all ranks.

Reply:
Ok, I will remove the usage of dminfo within `ozini`.  All ranks will get their own file unit (new behavior) and read the same file (same as existing behavior).  I think I was trying to follow `radaeini` with the rank checks, but I never added the communications.  Adding new comms is not my intention in this PR.

Reply:
I was going to add another commit that accomplishes my description above, but it seems like I could alternatively remove commit 22a14a9fd8efeadba9e7ca463389159c from history.  Doing that requires yet another rebase, which seems very frowned upon in all the reading I am doing about git histories.  Maybe it doesn't matter for a feature branch.  Is rebasing the preferred option from an MPAS-Atmosphere development standpoint?  The conflating factor is that this branch was originally stemmed from an old `develop` commit several years ago.  So removing old commits requires going through all the new develop commits that has to be merged into this branch.  If you would rather that I rebase, what is the easiest way to do it?  Create another branch and PR?

Reply:
I gave it a try by checking out the commit before the one that needed to be removed, then doing a force push with `git push -f origin HEAD:atmosphere/unformatted_io`

Reply:
In general, rebasing feature branches is not frowned upon. The problem with rebasing a branch that many people might have in their forks is that your rebased branch then becomes inconsistent (history-wise) with everyone else's. But, if the intent is to soon merge that branch into 'develop', this is probably not a problem.